src/assignments/launch-checklist.rst

```diff
@@ -0,0 +1,78 @@
+Assignment #5: Launch Checklist Form
+====================================
+
+Using our knowledge of forms, the DOM, and HTTP, our favorite space shuttle program has asked us to create a quick launch checklist.
+We have four fields that need to be filled out with vital information: the pilot's name, the co-pilot's name, the fuel levels, and the weight of the cargo.
+
+Our pilot, Chris, and their co-pilot, Blake, have been hard at work securing the cargo and filling the shuttle tank. All we need to do is use validation to insure that we have all of the info for the space shuttle program and make sure no one prematurely launches the shuttle.
```

Comment:
Word replacements:
a) "... and the co-pilot..."
b) "ensure" instead of "insure".

Reply:
Done! Thanks!